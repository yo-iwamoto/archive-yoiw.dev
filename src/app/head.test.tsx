import Head from './head';
import { render } from '@testing-library/react';

describe('Head', () => {
  it('renders unchanged', () => {
    const { getByText, asFragment } = render(<Head />);

    expect(getByText('yoiw.dev')).toBeDefined();
    expect(asFragment()).toMatchSnapshot();
  });
});
