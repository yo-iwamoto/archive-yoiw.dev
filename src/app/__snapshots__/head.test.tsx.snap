// Vitest Snapshot v1

exports[`Head > renders unchanged 1`] = `
<DocumentFragment>
  <title>
    yoiw.dev
  </title>
</DocumentFragment>
`;
